README.md

```diff
@@ -7,11 +7,10 @@
 npm install
 
 ## build
-npm run eslint
-npm run build
+cd addon/webextension
 
 ## build and run
-npm run firefox
+web-ext run
```

Comment:
We may need to add `npm install web-ext -g` to some list of prerequisites.

Conversely, add **web-ext** to devDependencies, and then we can do some package.json scripts to run the `web-ext run` and `web-ext lint` commands from the locally install web-ext module.

Reply:
+1